src/CodeEditor.tsx

```diff
@@ -13,6 +13,44 @@ export interface ICodeEditorProps {
   onChange?: (code: string) => any;
 }
 
+const defaultSupport = {
+  css: 'CSS',
+  javascript: 'JavaScript',
+  markup: 'Markup',
+  clike: 'C-like',
+};
+
+const optionalSupport = {
+  // cpp: 'C++',
+  // csharp: 'C#',
+  // go: 'Go',
+  // html: 'HTML',
+  // http: 'HTTP',
+  // java: 'Java',
+  // json: 'JSON',
+  // jsx: 'JSX',
+  // markdown: 'Markdown',
+  // objectivec: 'Objective-C',
+  // perl: 'Perl',
+  // php: 'PHP',
+  // python: 'Python',
+  // ruby: 'Ruby',
+  // scala: 'Scala',
+  // bash: 'Bash',
+  // swift: 'Swift',
+  // yaml: 'YAML',
+};
+
+/**
+ * List of supported languages: https://prismjs.com/#languages-list
+ */
+export const supportedLanguages = {
+  ...defaultSupport,
+  ...optionalSupport,
+};
+
+export const defaultLanguage = 'javascript';
```

Comment:
Could be a default prop.

Reply:
Not sure I get this. It's not a default export :) it's a hint for users of CodeEditor component which language to fallback by default

Reply:
I mean it could be default prop for the CodeEditor component, so that when you don't pass any language, then the default one is used.

Reply:
ooh :) Ok good point, but the exported value can be useful too (for selecting a default value in a dropdown for instance)

Reply:
Yeah, I am fine with exporting it :) Just wanted to have a default one too.
Make sure to adjust ICodeEditorProps to reflect the default prop.